### DISCUSSION TITLE
What are the NTRBoot firms for?

### DISCUSSION THREAD

**Discussion by daemonspudguy** [d0k3/GodMode9#725] [Q&A] [answered]
Are they so I can flash GM9 to my flashcart? If so, how?

**Comment by GerbilSoft** [comment, marked as answer]
Those are in fact for flashing to a compatible NDS flashcart for use with ntrboothax.

You can use ntrboot_flasher: https://github.com/ntrteam/ntrboot_flasher

I'll note that the current version, v0.5.0, wasn't able to detect my R4i-SDHC that already had a previous version of GM9 flashed to it. I was able to use v0.3.1 to upgrade it to the latest version, though.
